### DISCUSSION TITLE
JSBSim on Windows - Unable to Install - Need Help

### DISCUSSION THREAD

**Discussion by Nishi-Mishra** [JSBSim-Team/jsbsim#930] [Q&A]
Hi I am trying to install JSBSim on windows. I ran the setup.exe from (https://github.com/JSBSim-Team/jsbsim/releases/download/v1.1.13/JSBSim-1.1.13-setup.exe). It created a directory and I saw all the files. There were two executables: jsbsim.exe & aeromatic.exe. The aeromatic.exe runs fine, but the jsbsim.exe flashes another window and doesn't do anything. I tried the pip install jsbsim and I see it in my package list, but in a script I cannot import jsbsim. It says ModuleNotFoundError: No module named 'jsbsim'. 

I also tried pip install with wheels from https://pypi.org/project/JSBSim/#files, the file is called [JSBSim-1.1.13-pp39-pypy39_pp73-win_amd64.whl ](https://files.pythonhosted.org/packages/19/fb/75c3f4e8dbb181e83812fb58568e2e5cf31829fe8dfe39617b928f2a86e1/JSBSim-1.1.13-pp39-pypy39_pp73-win_amd64.whl) 

I would appreciate if someone can tell me what to do next to get JSBSim working on windows 11 either from the command prompt or python. And what is the output supposed to be.

**Comment by agodemar** [comment]
@Nishi-Mishra To start using this JSBSim software, you really need to go through its documentation. My advice is to start from: https://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf
Then you might want to go and read this: https://jsbsim-team.github.io/jsbsim-reference-manual/

**Comment by seanmcleod** [comment]
> but the jsbsim.exe flashes another window and doesn't do anything.

I'm assuming you're double-clicking on `JSBSim.exe` in an Explorer window and then seeing a console window open briefly and the close?

JSBSim.exe isn't a GUI application, so open a console window and run `JSBSim.exe` in a console window so you can see the output from JSBSim, for example you can see the output you get if you run it with no command line arguments, which is what is happening if you run it by double-clicking on it in an Explorer window.

![image](https://github.com/JSBSim-Team/jsbsim/assets/824856/2c6a68ac-bd3f-4256-bdd3-7c7bbee0929b)

As @agodemar says you'll also need to go through some of the documentation.
